isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -275,7 +275,7 @@ namespace isobus
 			                                         0xFF,
 			                                         0xFF,
 			                                         0xFF };
-		return queue_command(buffer, CANIdentifier::CANPriority::PriorityLowest7);
+		return queue_command(buffer);
```

Comment:
This is one of those implicit conversions that is simply incorrect. The second parameter is a `boolean`, while the priority enum represents `uint8_t`